src/Transporter/ImportFromPoint.php

```diff
@@ -57,6 +57,7 @@ public function import(
         $task->setAddress($address);
         $task->setComments($point->getComments());
         $task->setMetadata('imported_from', $imported_from);
+        $task->setMetadata('barcode', $point->getId());
```

Comment:
will it work if we have one barcode per parcel? in this case the barcode should be saved on `TaskPackage` object

Reply:
There is only the possibility for "one barcode per task" in the EDIFACT specification. So in this context no issue